python/ray/rllib/env/base_env.py

```diff
@@ -66,12 +66,15 @@ class BaseEnv(object):
     """
 
     @staticmethod
-    def to_base_env(env, make_env=None, num_envs=1):
+    def to_base_env(env, make_env=None, num_envs=1, remote_envs=False,
+                    entangled_envs=False):
         """Wraps any env type as needed to expose the async interface."""
         if not isinstance(env, BaseEnv):
             if isinstance(env, MultiAgentEnv):
+                # NOTE: Probably should handle remote / entangled envs in
+                #       _MultiAgentEnvToAsync as well
```

Comment:
Can we raise `NotImplementedError` here if those options are enabled?

Reply:
Done.